test/e2e/validation_test.go

```diff
@@ -561,8 +560,11 @@ func (tc *testContext) waitForValidWindowsServicesConfigMap(cmName string) (*cor
 		}
 		// Here, we've retreived a ConfigMap but still need to ensure it is valid.
 		// If it's not valid, retry in hopes that WMCO will replace it with a valid one as expected.
-		_, err = servicescm.Parse(configMap.Data)
-		return err == nil, nil
+		data, err := servicescm.Parse(configMap.Data)
+		if err != nil || data.ValidateRequiredContent() != nil {
```

Comment:
Looks like  `data.ValidateRequiredContent()` should be part of `servicescm.Parse(...)`. Thoughts?